entitlements.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>com.apple.security.cs.allow-unsigned-executable-memory</key>
```

Comment:
Do we need this? If so, why?

Reply:
This is the minimal requirement to pass notarisation with electron :) If we don't use that it will not succeed

Reply:
The Apple docs mention:
> In rare cases, an app might need to override or patch C code, use the long-deprecated NSCreateObjectFileImageFromMemory (which is fundamentally insecure), or use the DVDPlayback framework. Add the Allow Unsigned Executable Memory Entitlement to enable these use cases. Otherwise, the app might crash or behave in unexpected ways.
> ([Source](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_cs_allow-unsigned-executable-memory))

Is it safe to set this flag?

Reply:
It's as safe as Electron is 😅 I'll try to remove it to see if it still work 😄

Reply:
Well instantly crashed :(